src/patterns/exit-the-page-quickly/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Exit the page quickly
```

Comment:
From @davidc-gds:

> I think we usually go with ‘a’ instead of ‘the’?
> So, ‘Exit a page quickly’
> Would match ‘Start using a service’ and others.

Reply:
Reckon I'll do this, note to self:

- [ ] Will also need a change all mentions of the pattern.

Reply:
```suggestion
title: Exit a page quickly
```